### DISCUSSION TITLE
Is 0.9.x still a milestone?

### DISCUSSION THREAD

**Discussion by mattnelson** [eclipse-sisu/sisu-project#148] [General]
This may be a question for @cstamas and the Maven team, but since @cstamas is active in both repos and the "fix" would be started here, I'll start here.

The latest milestone of sisu.inject is now pulled into the latest non-milestone 3.9.x of maven. When our dependency uplift automation ran it recommended maven but not the version of sisu used by that version of maven because the milestone suffix excluded[2] it from consideration.

Is 0.9.x still a milestone?
If not, can I request a release of `0.9.0`?
If yes, what is left to promote out of milestone?

[1] https://github.com/apache/maven/pull/1286
[2] https://www.mojohaus.org/versions/versions-maven-plugin/examples/display-dependency-updates.html

**Comment by cstamas** [comment]
Sisu is really just a "runtime" dependency, unsure why you need to depend on it. All you need is javax.inject and presence of sisu index. The 1.0.0 is planned to happen soon though, cannot say exact date when, but "soon".

@mcculls ping

**Comment by mattnelson** [comment]
> Sisu is really just a "runtime" dependency, unsure why you need to depend on it

In our use case, we are using [mima](https://github.com/maveniverse/mima) which pulls maven(and all the transitives) into our dependency tree. Then we explicitly manage all versions of the dependency tree. What I only just connected the dots on is that when I use mima's `standalone-static` I can exclude the sisu's deps from maven-core.

**Comment by cstamas** [comment]
Oh true, basically sisu upgrade was asked by users doing maven components in Java 17. If you stay at Java 11 (or you do not create Maven plugins, extensions at all) you should be fine on Sisu 0.3.5, as whole Maven3 ecosystem is still Java 8.

**Comment by slachiewicz** [comment]
before 1.0, it would be good to add support for jakarta
